### ISSUE TITLE
Clear static content when enabling new module from CLI

### ISSUE THREAD

**Issue by markoshust** [magento/magento2#1711]
when running module:enable:

```
root@1ade1c331026:/var/www/html# ./bin/magento module:enable Foo_Bar
The following modules have been enabled:
- Foo_Bar

To make sure that the enabled modules are properly registered, run 'setup:upgrade'.
Cache cleared successfully.
Generated classes cleared successfully.
Alert: Generated static view files were not cleared. You can clear them using the --clear-static-content option. Failure to clear static view files might cause display issues in the Admin and storefront.
```

The alert message seems problematic:
`Alert: Generated static view files were not cleared. You can clear them using the --clear-static-content option. Failure to clear static view files might cause display issues in the Admin and storefront.`

If there are issues if the `--clear-static-content` param isn't passed in, shouldn't it be the default option when enabling a module?

**Comment by mbrinton01** [comment]
@markoshust decision was made to make this optional by default, please see the following issue for background: https://github.com/magento/magento2/issues/1108.  Do you think the alert message could be improved?

**Comment by markoshust** [comment]
Thank you, I was looking for related ticket on this as I was sure there was one but I couldn't find it.

The responses in that ticket make sense. Right now, the "Alert" seems like a critical issue (I believe it's highlighted in red with CLI so it seems like a Fatal message). The "Failure to clear static view files might cause display issues in the Admin and storefront." also seems like a critical issue. I don't think it's exactly clear when this should be cleared, and when it doesn't need to be cleared.

I think improving the message makes sense, and perhaps change it to an `Info` instead of an `Alert` message type (and remove red background). Perhaps something along the lines of "Info: To clear generated static view files, use the --clear-static-content option. Some modules may require static assets to be cleared, please contact your module developer for more details".

I'm not sure if that verbiage is correct, but I think it might make sense to change it from critical/alert message to more of a passive info message.

**Comment by mbrinton01** [comment]
Thank you for your feedback, I will review with the team and provide updates here.

**Comment by xcomSteveJohnson** [comment]
Suggest a minor tweak:

"Info: Use the optional --clear-static-content option to clear generated static view files. Some modules might require static files to be cleared. Contact your module vendor for more details".

(Although to be realistic, is anyone going to stop and contact the module vendor?)

**Comment by markoshust** [comment]
Good point about the contacting vendor. Can probably leave that last line out.

**Comment by mbrinton01** [comment]
Tracking internally as MAGETWO-42025

**Comment by mbrinton01** [comment]
@markoshust this has been completed but not yet pushed to GitHub repo.  Closing the issue - feel free to re-open if you don't see it in the near future or have more questions.

**Comment by magento-engcom-team** [comment]
Hi @markoshust. Thank you for your report. 
The issue has been fixed in magento/magento2#29435 by @jmonteros422 in 2.4-develop branch
Related commit(s):
 - [11d0a351c4ad83637620c9b7ab00ef68b028b7fa](https://github.com/magento/magento2/commit/11d0a351c4ad83637620c9b7ab00ef68b028b7fa)
 - [25133c56cf0e0390c11399b78cbe3fb426fdd1be](https://github.com/magento/magento2/commit/25133c56cf0e0390c11399b78cbe3fb426fdd1be)
 - [c72447ff51cf095595bbe47b67afe0d45d5b0941](https://github.com/magento/magento2/commit/c72447ff51cf095595bbe47b67afe0d45d5b0941)
 - [73d032fb4997c142166698e209907f6e49f524b0](https://github.com/magento/magento2/commit/73d032fb4997c142166698e209907f6e49f524b0)
 - [36427dc5802f140edc0d6bdbaa1d78534ff3d7ad](https://github.com/magento/magento2/commit/36427dc5802f140edc0d6bdbaa1d78534ff3d7ad)

The fix will be available with the upcoming 2.4.1 release.
